packages/relay-compiler/bin/RelayCompilerBin.js

```diff
@@ -122,6 +128,9 @@ function getRelayFileWriter(baseDir: string) {
 function getSchema(schemaPath: string): GraphQLSchema {
   try {
     let source = fs.readFileSync(schemaPath, 'utf8');
+    if (path.extname(schemaPath) === '.json') {
+      source = printSchema(buildClientSchema(JSON.parse(source).data));
+    }
```

Comment:
Ok. I knew about [`graphql-config`](https://github.com/graphcool/graphql-config) but not about get-graphql-schema. Thanks @sibelius